fiesta/apps/notifications/models/scheduled.py

```
@@ -0,0 +1,80 @@
+    content_object = GenericForeignKey("content_type", "object_id")
+
+    send_after = models.DateTimeField(verbose_name=_("send after"))
+    sent_at = models.DateTimeField(null=True, blank=True, verbose_name=_("sent at"))
+    cancelled_at = models.DateTimeField(null=True, blank=True, verbose_name=_("cancelled at"))
+
+    class Meta:
+        verbose_name = _("scheduled notification")
+        verbose_name_plural = _("scheduled notifications")
+        indexes = [
+            models.Index(fields=["send_after", "sent_at", "cancelled_at"]),
+        ]
```

Ruff (0.15.4)

[warning] Mutable default value for class attribute

(RUF012)